src/main/java/uk/gov/hmcts/reform/sscs/service/LetterUtils.java

```diff
@@ -41,18 +34,6 @@ public static Address getAddressToUseForLetter(NotificationWrapper wrapper, Subs
         }
     }
 
-    public static String getFilename(NotificationWrapper wrapper) {
```

Comment:
What is the reason the filename has been removed.

Reply:
None of it was used.